### PR TITLE
[AUTO] Show the detailed failure message when AUTO load network failed

### PR DESCRIPTION
**Pull request by WeldonWangwang**
### Details:
 - Show the detailed failure message when AUTO load network failed

### Tickets:
 - *CVS-101342, CVS-90102*

### CODE REVIEW

File: src/plugins/auto/auto_schedule.cpp [hunk 1]
```diff
@@ -536,13 +536,16 @@ void AutoSchedule::WaitFirstNetworkReady() {
...
     for (int i = CONTEXTNUM - 2; i >= 0; i--) {
         if (_loadContext[i].isEnabled) {
-            LOG_ERROR_TAG("load failed, %s", _loadContext[i].errMessage.c_str());
+            result << _loadContext[i].errMessage.c_str();
+            result << "; ";
         }
     }
-    IE_THROW() << GetLogTag() << "load all devices failed";
+    IE_THROW() << "[" << GetLogTag() << "] " << result.str();
 }
```
Comment (songbell) [inline src/plugins/auto/auto_schedule.cpp:549]: can we add test?

Comment (WeldonWangwang) [reply src/plugins/auto/auto_schedule.cpp:549]: Sure

Comment (WeldonWangwang) [reply src/plugins/auto/auto_schedule.cpp:549]: Added.

File: src/plugins/auto/plugin_config.cpp [hunk 2]
```diff
@@ -4,7 +4,6 @@
 #include "utils/plugin_config.hpp"
 
 namespace MultiDevicePlugin {
-const std::set<std::string> PluginConfig::_availableDevices = {"AUTO", "CPU", "GPU", "TEMPLATE", "NVIDIA", "VPUX", "MULTI", "HETERO", "mock"};
```
Comment (wangleis) [inline src/plugins/auto/plugin_config.cpp:7]: Please add this change into PR details.

**Review by wangleis — APPROVED** [review]

File: src/plugins/auto/plugin_config.cpp (hunk 2 above)
Comment (WeldonWangwang) [reply src/plugins/auto/plugin_config.cpp:7]: Added.

**Pull request merged**